src/com/lftechnology/java_training/jan5/ValueSearchFromArray.java

```diff
@@ -0,0 +1,109 @@
+package com.lftechnology.java_training.jan5;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/**
+ * Search for a specific value in an array.
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class ValueSearchFromArray {
+	private static final Logger LOGGER = Logger.getLogger(ValueSearchFromArray.class.getName());
+	private static Scanner input = new Scanner(System.in);
+	private static final int EXIT_KEY = 3;
+
+	public static void main(String[] args) {
+		// creates new object to search value from array
+		SearchValue searchValue = new SearchValue();
+		int choice = 0;
+		try {
+			do {
+				LOGGER.info("Enter your choice:\n 1. store value in array" + "\n 2. enter value to search from array ");
+				choice = input.nextInt();
+
+				switch (choice) {
+				case 1:
+					searchValue.getArray();
+					break;
+				case 2:
+					searchValue.searchValueInArray();
+					break;
+
+				default:
+					break;
+				}
+			} while (!(choice == EXIT_KEY));
+		} finally {
+			input.close();
+		}
+
+	}
+}
+
+/**
+ * This class ask user to enter value to store in array when they finished adding then asked user to enter value to be searched in array and
+ * display whether the value is in array or not
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+class SearchValue {
+	private String valueToSearch = "";
+	private String[] arrayOfValues = new String[10];
+	private int countValuesInArray = 0;
+	private Scanner input = new Scanner(System.in);
+	private final static Logger LOGGER = Logger.getLogger(SearchValue.class.getName());
+	private final String Y = "y";
+	private final String YES = "yes";
```

Comment:
I have seen some of the final String in almost every class which as in fact Constants. Separate Constant class is recommended which will hold all the contants.